exercises/grade-school/Tests/GradeSchoolTests/GradeSchoolTests.swift

```diff
@@ -20,12 +19,6 @@ private extension XCTestCase {
         XCTAssert(sameCollection(collectionS, collectionA))
     }
 
-    func XCTAssertEqualCollection (_ collectionA1: [String]?, _ collectionA2: [String]? ) {
```

Comment:
This needs to go back in to accommodate array to array comparisons https://github.com/exercism/xswift/issues/258